openqaoa/problems/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .knapsack import Knapsack, SlackFreeKnapsack
+from .maximumcut import MaximumCut
+from .minimumvertexcover import MinimumVertexCover
+from .numberpartition import NumberPartition
+from .shortestpath import ShortestPath
+from .tsp import TSP
+from .converters import FromDocplex2IsingModel
+from .qubo import QUBO
```

Comment:
Maybe format it all with https://github.com/psf/black

Reply:
Pushing commit